<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryForm</class>
 <widget class="QWidget" name="HistoryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>History</string>
  </property>
  <widget class="QListWidget" name="sessionList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>256</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="messageDisplay">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>371</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="contentField">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>70</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Content</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nickname</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nicknameField">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>120</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeStart">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>220</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEnd">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>270</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>220</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>270</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>170</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hostField">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>170</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
